src/Controls/src/Core/Handlers/Items/iOS/ItemsViewLayout.cs

```diff
@@ -104,9 +104,9 @@ protected virtual void HandlePropertyChanged(PropertyChangedEventArgs propertyCh
 			}
 		}
 
-		internal virtual bool UpdateConstraints(CGSize size)
+		internal virtual bool UpdateConstraints(CGSize size, bool forceUpdate = false)
 		{
-			if (size.IsCloseTo(_currentSize))
+			if (size.IsCloseTo(_currentSize) && !forceUpdate)
```

Comment:
When used as `ContentPage.Content = new CollectionView()` the `_currentSize` never changes so this code path was not hit on the first `LayoutSubviews`, leading to wrong `preferredAttributes` computation and bad cell positioning.
You can verify this on `main` by testing out `Issue13203` where the text is being positioned in the middle behind the safe area instead of being left aligned.